test/NuGet.Clients.Tests/NuGet.VsExtension.Test/VSVersionHelperTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using EnvDTE;
+using Moq;
+using NuGet.PackageManagement.VisualStudio;
+using Xunit;
+
+namespace NuGet.VsExtension.Test
+{
+    internal class VSVersionHelperTests
```

Comment:
FYI, xUnit did not run these tests.  xUnit will not find tests in an `internal` class.  The class must be public.

You can prove this to yourself by temporarily modifying one of these tests to fail.